api/internal/handler/consumer/consumer_test.go

```diff
@@ -19,6 +19,10 @@ package consumer
 
 import (
 	"encoding/json"
+	"fmt"
+	"github.com/shiningrush/droplet/data"
```

Comment:
I think it's better to keep the convention about `import`: the first part is all standard libs, the second part is your local packages in your go module, and the last part is all third party packages.